doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -70,25 +69,6 @@ The following are the common requirements to use coexistence based on the MPSL C
    When using the nRF5340, apply steps 1 and 2 only to the network core.
    See :ref:`ug_multi_image`.
 
-.. _ug_radio_coex_bluetooth_only_based:
-
-MPSL provided Bluetooth-only coexistence
-========================================
-
-The MPSL-provided Bluetooth-only coexistence can be used only with the :ref:`ug_radio_coex_bluetooth_only_1wire` implementation.
-
-
-It is based on :ref:`nrfxlib:bluetooth_coex` provided by the :ref:`nrfxlib:mpsl` (MPSL) library.
-
-
-Supported implementations
-*************************
-
-The following are the implementations supported by the MPSL-provided Bluetooth-only coexistence.
-
-.. note::
-   Do not enable Wi-Fi coexistence on the nRF5340 SoC in conjunction with Coded Phy and FEM, as this can lead to undefined behavior.
```

Comment:
This is still a valid note but it seems it was misplaced in the docs, see https://github.com/nrfconnect/sdk-nrf/blob/a1e6032e852291542fe8bb78112cbc5dc0b038e6/subsys/mpsl/cx/Kconfig#L31

Reply:
moving it to "Coexistence based on MPSL CX API"